frontend/app/generateUserDetails.js

```diff
@@ -33,12 +33,20 @@ export function enableEndTurn() {
 
 /**
  * Callback to check user rolls and enable end turn.
- *
+ * Enables roll-dice if a double is rolled.
  * @param {XMLHttpRequest} req1 response.
  */
 export function successCallback(req1) {
```

Comment:
Is this needed? There's a function already implemented which handles calling roll dice and compares the values. Couldn't you just get rid of the JSONSend({......}); in the function rollDice() and replace it with updateRollButton(); and remove the successCallBack function?